ddtrace/llmobs/_evaluators/ragas/faithfulness.py

```diff
@@ -139,7 +139,7 @@ def __init__(self, llmobs_service):
             raise NotImplementedError("Failed to load dependencies for `ragas_faithfulness` evaluator") from e
         finally:
             telemetry_writer.add_count_metric(
-                namespace=TELEMETRY_APM_PRODUCT.LLMOBS,
+                namespace=TELEMETRY_APM_PRODUCT.LLMOBS.value,
```

Comment:
You can also make the enum inherit from `str` to avoid the `.value`
```python
class MyStrEnum(str, Enum):
    ...
```

Reply:
Ah thanks - i see this [PR](https://github.com/DataDog/dd-trace-py/commit/c2eb5eea5644135c4aff4d9e9add232531214b35) was merged which implements that, so i can actually close this PR